client/components/Item/PersonList.vue

```diff
@@ -8,12 +8,13 @@
         :to="getItemDetailsLink(item, 'Person')"
       >
         <v-list-item-avatar>
-          <v-img
-            v-if="item.PrimaryImageTag"
-            ref="personImg"
-            :src="getImageUrl(item.Id)"
-          />
-          <v-icon v-else class="grey darken-3">mdi-account</v-icon>
+          <v-avatar color="card">
+            <blurhash-image :item="item">
+              <template #placeholder>
+                <v-icon size="16">mdi-account</v-icon>
+              </template>
```

Comment:
Because this doesn't use BaseItemDto, but BaseItemPerson.

Reply:
@MrTimscampi Shouldn't we then instruct BlurhashImage to handle those? Right now they don't have a blurhash as well because this. As you moved everything to a single function in imageHelper I guess this should be possible without much hassle.

I'd rather we get it merged without this as the placeholder mess is very difficult to maintain and having everything centered (like image rescaling on resize, the fallbacks and so on) in blurhashimage turned out to be great in the long term (this PR would have been massive for instance :P)